app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -161,6 +162,32 @@ object BitcoindRpcBackendUtil extends Logging {
     }
   }
 
+  def createDLCWalletWithBitcoindCallbacks(
+      bitcoind: BitcoindRpcClient,
+      wallet: DLCWallet)(implicit system: ActorSystem): DLCWallet = {
+    // Kill the old wallet
+    wallet.stop()
```

Comment:
I don't think this is necessary now. This is a noop . All wallet resources were moved into `WalletAppConfig`

https://github.com/bitcoin-s/bitcoin-s/blob/4c1b5e345242afa6c1b5a9298f0c9cced94fa73f/wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala#L156